flake.nix

```diff
@@ -12,7 +12,8 @@
     # Use upstream when https://github.com/nlewo/nix2container/pull/82 is merged
     n2c.url = "github:shlevy/nix2container/no-Size-on-dir";
 
-    marlowe-plutus.url = "github:input-output-hk/marlowe-plutus";
+    # Version of marlowe-plutus supporting aarch64-linux (TODO: Merge on main)
```

Comment:
Cosmetics. We should probably merge that change into `marlowe-plutus` first. Tag a new patch release use that release number here instead of a hash of a commit.